packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -82,15 +82,20 @@ def get_measurement_service_stub(
     return v2_measurement_service_pb2_grpc.MeasurementServiceStub(channel)
 
 
-def get_all_registered_measurement_service_classes(discovery_client: DiscoveryClient) -> List[str]:
+def get_all_registered_measurement_info(
+    discovery_client: DiscoveryClient,
+) -> Tuple[List[str], List[str]]:
     """Returns the service classes of all the registered measurement services."""
     registered_measurement_services = discovery_client.enumerate_services(
         _V2_MEASUREMENT_SERVICE_INTERFACE
     )
     measurement_service_classes = [
         measurement_service.service_class for measurement_service in registered_measurement_services
     ]
-    return measurement_service_classes
+    measurement_display_names = [
+        measurement_service.display_name for measurement_service in registered_measurement_services
```

Comment:
we are retrieving both the display name and service class from the same object. So, we can use a single for loop to fetch both the information.

Reply:
We can get them as `tuple` using a single for loop. By doing so, we need to separate service class and display names using another for loop in callers. So I think returning them as separate list here should be fine.